src/Equinox/Category.fs

```diff
@@ -20,24 +20,22 @@ type ICategory<'event, 'state, 'context> =
                    * originToken: StreamToken * originState: 'state * events: 'event[]
                    * CancellationToken -> Task<SyncResult<'state>>
 
-// Low level stream operations skeleton; base type for Store-specific Category types
+module StreamName =
+    let render (cat: string) (sid: string): string =
+        FsCodec.StreamName.create cat (FsCodec.StreamId.Elements.trust sid) |> FsCodec.StreamName.toString
+
 namespace Equinox
 
 open Equinox.Core.Tracing
 
 /// Store-agnostic baseline functionality for Load and Syncing a Category of 'event representations that fold to a given 'state
+/// Provides access to the low level store operations used for Loading and/or Syncing updates via the Decider
+/// (Normal usage is via the adjacent `module Decider` / `Stream.Resolve` helpers)
 [<NoComparison; NoEquality>]
-type Category<'event, 'state, 'context>(categoryName, resolveStream) =
-
-    /// Stores without custom routing for categoryName/streamId to Table/Container etc use this default impl
-    new(categoryName, inner) = Category(categoryName, fun streamId -> struct (inner, Core.StreamName.render categoryName streamId))
-
-    /// Provides access to the low level store operations used for Loading and/or Syncing updates via the Decider
-    /// (Normal usage is via the adjacent `module Decider` / `Stream.Resolve` helpers)
+type Category<'event, 'state, 'context>(categoryName, inner: Core.ICategory<'event, 'state, 'context>) =
     member _.Stream(log: Serilog.ILogger, context: 'context, streamId: string) =
```

Comment:
```suggestion
    member _.Stream(log: Serilog.ILogger, context: 'context, streamId: FsCodec.StreamId) =
```

Not sure if possible or not

Reply:
Good call, thanks!
StreamName manipulation for MDB snapshot (grep `let inline streamName`) is a big ugly, but no special cases or direct UMX usage anywhere now